proto/api/v1/resource_collection_service.proto

```diff
@@ -21,6 +21,7 @@ message ListCollectionSelectorsResponse {
 message GetCollectionRequest {
     message Options {
         bool with_matches = 1;
+        Pagination pagination = 2;
```

Comment:
This is a bit ambiguous. Is this pagination for collections or the matched deployments? Either a nested object or matches_pagination or something like that is nicer.